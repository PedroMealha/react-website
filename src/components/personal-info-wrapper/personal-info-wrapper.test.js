import { shallow } from "enzyme";
import PersonalInfoWrapper from "./personal-info-wrapper.component";

describe("PersonalInfoWrapper component", () => {
	it("expect to render", () => {
		const wrapper = shallow(<PersonalInfoWrapper />)
		expect(wrapper).toMatchSnapshot();
	});
});
