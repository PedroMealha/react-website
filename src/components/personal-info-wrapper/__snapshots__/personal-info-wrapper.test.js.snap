// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonalInfoWrapper component expect to render 1`] = `
<div
  className="personal-info-wrapper"
>
  <div
    className="personal-info"
  >
    <div
      className="location"
    >
      <ul>
        <li>
          <h4>
            Location
          </h4>
          <p>
            Portugal
          </p>
        </li>
        <li>
          <h4>
            City
          </h4>
          <p>
            Lisbon
          </p>
        </li>
        <li>
          <h4>
            Experience
          </h4>
          <span>
            6+
          </span>
        </li>
      </ul>
    </div>
    <Divider />
    <div
      className="languages-wrapper"
    >
      <h4>
        Languages
      </h4>
      <div
        className="languages"
      >
        <LanguageCard
          language="Portuguese"
          percent="100"
        />
        <LanguageCard
          language="English"
          percent="95"
        />
      </div>
    </div>
    <Divider />
    <div
      className="skills-wrapper"
    >
      <Skills />
    </div>
  </div>
</div>
`;
